src/components/MainSection.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Components from '/images/image3.png';
+import Singleway from '/images/image4.png';
+import JSX from '/images/image5.png';
+
```

Comment:
in here you can fake data like server creating a new object like 

```
const arrayCard = [
{
title: "sasa",
image: " url of image",
text: "my text",
}
]
```
```

Reply:
Thank you